language-support/java/bindings-rxjava/src/test/scala/com/daml/ledger/rxjava/grpc/CommandSubmissionClientImplTest.scala

```diff
@@ -36,7 +37,7 @@ class CommandSubmissionClientImplTest
   it should "timeout after deadline exceeded" in {
```

Comment:
There is still no test checking that the error is fixed.

Reply:
I think a good test would be to make 2 command submissions on the same instance of CommandSubmissionClientImpl. 

1. The 1st (time > timeout) should fail (similar to existing deadline test case).
2. The 2nd (time < timeout) should pass OK.

However, I am completely new to Scala, so would like to seek your help.

Reply:
I opened a PR on your repo [here](https://github.com/dparmar/daml/pull/1). If you have a look at that and merge it, those changes should be reflected here (for some reason the code formatting tool made the diff larger than I expected, apologies for that).

Reply:
Thanks a lot for your help! I just merged it.